src/components/data_comps/desp/cime_config/namelist_definition_desp.xml

```diff
@@ -57,7 +57,7 @@
       Level of debug output, 0=minimum, 1=normal, 2=more, 3=too much (default: 0)
     </desc>
     <values>
-      <value>0</value>
+      <value>1</value>
```

Comment:
This isn't the same info_debug used by the coupler? I don't think that it should have the same name.  This is a cause for confusion as I was changing info_dbug in the xml and thought that I was toggling this value.   Why do we need this flag at all?  It seems that all of these messages written from mastertask to the desp log are appropriate.